src/main/java/org/datadog/jenkins/plugins/datadog/DatadogUtilities.java

```diff
@@ -677,9 +680,16 @@ private static boolean isWithinIPv4OctetRange(int number) {
 
     public static Map<String, Set<String>> getComputerTags(Computer computer) {
         Set<LabelAtom> labels = null;
-        try {
-            labels = computer.getNode().getAssignedLabels();
-        } catch (NullPointerException e) {
```

Comment:
Can we log a warning instead if this is the case? It's ok if we can't get tags for computers

Reply:
Done.